README.md

```diff
@@ -112,19 +198,6 @@ variable. Using "/" as the context path means that we can have certain
 resources (such as health checks) with a known, static name while others
 can vary based on an environment variable given to the pod.
 
-### Static Endpoints
```

Comment:
Instead of deleting this section entirely, do we want to add something like

```
Due to constraints imposed by Clowder, the Spring Boot Actuator endpoints are served off the root context on port 9000.  E.g. `localhost:9000/health` or `localhost:9000/hawtio`.
```

Reply:
Actually yeah, I think I was moving some things around and dropped this section unintentionally. Let me restore it w/ some edits.